File: codewars/8kyu/what-is-between/dduck-what-is-between.py
# https://www.codewars.com/kata/what-is-between/train/javascript

between = lambda a, b: list(range(a, b + 1))

print(between(1, 4))
# [1, 2, 3, 4];

print(between(-2, 2))
# [-2, -1, 0, 1, 2];

print(between(1, 1))

File: codewars/8kyu/what-is-between/dduck-what-is-between.js
// https://www.codewars.com/kata/what-is-between/train/javascript

const between = (a, b) => Array(b - a + 1).fill().map((v, i) => a + i);

console.log(between(1, 4));
//?
//, [1, 2, 3, 4];

between(-2, 2);
//?
// [-2, -1, 0, 1, 2];

between(1, 1);
//?
